scripts/bump_abacus.sh

```diff
@@ -31,26 +31,14 @@ scriptDir=`pwd`
 cd /tmp
 rm -rf abacus
 gh repo clone git@github.com:dydxprotocol/v4-abacus.git abacus
-version=`cat abacus/v4_abacus.podspec | grep spec.version | grep -o '[0-9]*\.[0-9]*\.[0-9]*'`
+version=$(grep "^version = " abacus/build.gradle.kts | sed -n 's/version = "\(.*\)"/\1/p')
 echo "=================================="
 echo "Bumping to $version.... hang on..."
 
 
 
 echo "=================================="
-echo "Checking git tag v$version"
-
-cd abacus
-exist=`git tag | grep "v$version" || true`
-if [ ! -z "$exist" ]; then
-  echo "v$version exists"
-else
-  echo "Creating git tag v$version"
-  username=$(git config user.name)
-  git tag -a "v$version" -m "v$version created by $username"
-  git push --tags origin main
-fi
-
```

Comment:
this will be moved to [bitrise](https://dydx-team.slack.com/archives/C03M1B6CYVA/p1718996082976659), see message